Callback Added

BerkinAKKAYA opened the pull request:
`onInput` Callback Added

### Usage:

```js
<DatePicker onInput={date => { console.log(date) }} />
```

SharifClick commented:
Hi @BerkinAKKAYA thank you for your PR, 
I think there should be 2 type of callback we need to hook on that component. The PR you open is good logically, can you please change the callback name `onConfirmDate`.
Later i will add another callback `onDateChange` for every item change.
Thanks.

BerkinAKKAYA commented:
Hi @SharifClick, I renamed `onConfirmDate` and added `onDateChange` callback. Let me know if you had something else in mind please. I needed this in my own project so i added them according to my needs. Have a great day, thanks to you!

SharifClick commented:
Merged and released new version, 
Thanks for PR, 
Please follow readme. 
Have a great day.
